Cargo.lock

```diff
@@ -3412,9 +3412,9 @@ dependencies = [
 
 [[package]]
 name = "miniz_oxide"
-version = "0.8.0"
+version = "0.8.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "e2d80299ef12ff69b16a84bb182e3b9df68b5a91574d3d4fa6e41b65deec4df1"
+checksum = "4ffbe83022cedc1d264172192511ae958937694cd57ce297164951b8b3568394"
```

Comment:
why this update? fine to do it but maybe in a different PR?

Reply:
cargo automatically updates this, probably they yanked some version...

Reply:
this should never happen, cargo doesn't auto-update yanked versions.